common.lic

```diff
@@ -283,4 +298,11 @@ module DRC
       .map { |_name, properties| properties['abbrev'] }
       .each { |abbrev| fput("release #{abbrev}") }
   end
+  
+  def check_encumbrance
+    enc = nil
+    enc_msg = bput('encumbrance', 'Encumbrance : .*')
```

Comment:
You can get your current encumbrance (as a string) without sending a command via `DRStats.encumbrance`. You'll still have to map that, though.